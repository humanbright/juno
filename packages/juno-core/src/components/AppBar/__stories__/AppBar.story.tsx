import React, { ComponentProps } from 'react';

import { RcAppBar } from '@ringcentral/juno';
import {
  notControlInDocTable,
  notShowInDocTable,
  sortInDocTable,
} from '@ringcentral/juno-storybook';
import { Meta, Story } from '@storybook/react';

export default {
  title: 'ðŸš€ Cleanup Components/AppBar',
  component: RcAppBar,
  argTypes: {
    ...sortInDocTable<keyof AppBarProps>([]),
    ...notControlInDocTable<keyof AppBarProps>([]),
    ...notShowInDocTable<keyof AppBarProps>([]),
  },
} as Meta;

type AppBarProps = ComponentProps<typeof RcAppBar>;

export const AppBar: Story<AppBarProps> = ({ children, ...args }) => {
  return <RcAppBar {...args}>{children}</RcAppBar>;
};

AppBar.storyName = 'AppBar';

AppBar.args = {};

AppBar.argTypes = {
  ...notControlInDocTable<keyof AppBarProps>([]),
};

AppBar.parameters = {
  tags: [
    // {
    //   name: 'Spec',
    //   href: '',
    // },
    {
      name: 'Mui',
      href: 'https://material-ui.com/components/app-bar',
    },
    {
      name: 'Accessibility',
      value: '100%',
    },
  ],
};
