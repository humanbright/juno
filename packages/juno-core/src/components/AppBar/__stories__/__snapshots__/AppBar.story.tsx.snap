// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots ðŸš€ Cleanup Components/AppBar AppBar 1`] = `
.c0 {
  height: 56px;
  background-color: #FFFFFF;
  background: linear-gradient( to right, #066FAC, #509AC4 );
  border-bottom: 1px solid rgba(0,0,0,0.12);
  z-index: 1210;
}

<header
  className="MuiPaper-root MuiAppBar-root MuiAppBar-positionStatic MuiAppBar-colorPrimary c0 MuiPaper-elevation0"
/>
`;
